### ISSUE TITLE
I want to go to NEO DevCon! My email: xquentin_gille@hotmail.com

### ISSUE THREAD

**Issue by Dipx** [neo-project/neo.org#554]

**Comment by chenzhitong** [comment]
Write down a brief self-introduction and area of development you're good at. And write down links to GitHub blockchain projects that you are involved in.

**Comment by Dipx** [comment]
My name is Quentin and I'm from Belgium. I work as a developer since 5 years. My expertise field has always been the web development, and particularly the front end. I'm certified by microsoft as a javascript developer. And I really like new technologies and new framework (Like reactJs, angularJs 5, ...). I followed many online cursus with Todd Motto (AngularJs) and Kyle Simpson (EcmaScript 6).
I did not made any blockchain project. (Only a few local website to help myself manage my portfolio, like a website that analyse all addresses that I give it to, and give me back the balance of that address, the tokens and their valuations (Many blockchain supporteds). But I never posted it on github as it contains sensible information.
If you have any other questions, feel free to ask it :)
